release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1224,6 +1244,14 @@ This update corrects the accepted label specification so that the plug-in import
 
 * Previously, when reusing an existing statically keyed LUKS volume during provisioning, the encryption key was not correctly written to disk and Ignition would fail with a "missing persisted keyfile" error. With this fix, Ignition now correctly writes keys for reused LUKS volumes so that existing statically keyed LUKS volumes can be reused during provisioning. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2043296[*BZ#2043296*])
 
+* Previously, the `ostree-finalize-staged.service` failed while upgrading a {op-system-first} node to 4.6.17. To fix this, the sysroot code now ignores any irregular/non-symlink files in `/etc`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1945274[*BZ#1945274*])
```

Comment:
perhaps remove "the" before  `ostree-finalize-staged.service`

irregular or non-symlink